include_core/omrport.h

```diff
@@ -228,6 +228,8 @@
 #define OMRPORT_VMEM_MEMORY_MODE_COMMIT 0x00000008
 #define OMRPORT_VMEM_MEMORY_MODE_VIRTUAL 0x00000010
 #define OMRPORT_VMEM_MEMORY_MODE_SHARE_FILE_OPEN 0x000000200
+#define OMRPORT_VMEM_MEMORY_MODE_DOUBLE_MAP_AVAILABLE 0x000000800
+#define OMRPORT_VMEM_MEMORY_MODE_HUGE_PAGES 0x000000400
```

Comment:
OMRPORT_VMEM_MEMORY_MODE_HUGE_PAGES  actually means OMRPORT_VMEM_MEMORY_MODE_MMAP_HUGE_PAGES, right?

Reply:
Yes, will update it